src/components/ui/icons/TrashIcon/TrashIcon.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from "prop-types";
+import "@components/ui/icons/_iconBase.scss";
+
+function TrashIcon({ className, ...rest }) {
+    return (
+        <svg
+            xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 24 24"
+            fillRule="evenodd"
+            clipRule="evenodd"
+            className={className}
+            {...rest}
+        >
+            <path
+                strokeLinecap="round"
+                strokeLinejoin="round"
+                d="M19 24h-14c-1.104 0-2-.896-2-2v-16h18v16c0 1.104-.896 2-2 2m-9-14c0-.552-.448-1-1-1s-1 .448-1 1v9c0 .552.448 1 1 1s1-.448 1-1v-9zm6 0c0-.552-.448-1-1-1s-1 .448-1 1v9c0 .552.448 1 1 1s1-.448 1-1v-9zm6-5h-20v-2h6v-1.5c0-.827.673-1.5 1.5-1.5h5c.825 0 1.5.671 1.5 1.5v1.5h6v2zm-12-2h4v-1h-4v1z"
+            />
+        </svg>
```

Comment:
@sse-wtag Bhai, why aren't we moving this svg as per this [comment](https://github.com/sse-wtag/todo-app-react/pull/2#discussion_r1339807437)?

Reply:
@fza-wtag bhai, If I move SVGs and try to use them as a React component, either I need to use CRA or an external package (e.g. [SVGR](https://github.com/gregberge/svgr)). But I'm using Vite.js as my build tool, which doesn't allow this to happen. I could move the SVGs, import them, and use them as a source for the `img` element. But in this way, I will lose control to change the color in the future. So, that's why I followed this approach. Thank you.
[reference1](https://betterprogramming.pub/react-best-way-of-importing-svg-the-how-and-why-f7c968272dd9#:~:text=Here%E2%80%99s%20an%20image,Component%20(Inline%20SVG)), [reference2](https://www.sanity.io/guides/import-svg-files-in-react)